cammac60/MacRae-Photo
# Bem refactor

## What does this PR do? 

- Refactors all SASS files and component JSX to match BEM standards.
- Removes some unneeded CSS.

----------

### How to manually test: 

- `git fetch`
- `git checkout bem-refactor`
- `npm start`
- `npm test` to ensure all tests are passing.
- Open the site and go through every page to ensure that the styles are still working.

---------

### Tickets: 

- Resolves #60 

----------

### Notes: 

 None
